### ISSUE TITLE
HTML code appears in Lemma ID Gloss field

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#468]
* This is a returning issue, but I cannot find back the earlier issue.
* ASL example: https://aslsignbank.haskins.yale.edu//dictionary/gloss/2054/
* Julie: "First off, when I clicked on this entry, I saw the "a href..." text (see by "lemma ID gloss") and remembered seeing that before. I think it was from when I used { } in a lemma ID gloss. I removed it wherever I used it but it seems to have persisted (if that is even the problem)."
* Julie: "When I tried uploading this video, I got this report (see attached)." In attachment: KeyError at /video/upload. (video: https://www.dropbox.com/s/fuehzjw1lvsxyj6/THIRTEEN-YEARS-OLD.mp4?dl=0)

**Comment by Woseseltops** [comment]
I changed the title for clarity: the problem is not the hyperlink (that's what we want), but the fact that we see actual html displayed instead of interpreted by the browser.

Some more background info on what is happening:
* The html really ended up in the database (I could also see it in the admin).
* Once I figured that out, it was simply a matter of changing the field back to what it should have been, and now the problem for this particular gloss seems solved. A quick search in the admin showed this was the only gloss that had this problem.
* I can imagine that uploading videos for this gloss will be problematic, as the html might end up in the video name, causing all kinds of troubles. The video works now, in any case - I didn't even have to upload anything.

The big question of course is: how did the html end up there? The hyperlink is there in interpreted form on the normal detail view, and it is turned into normal text when you click the 'edit' button. Perhaps something went wrong there, perhaps during an earlier broken version of the feature?
